eng/common/TestResources/deploy-test-resources.yml

```diff
@@ -39,7 +39,7 @@ steps:
 
   - pwsh: |
       eng/common/scripts/Import-AzModules.ps1
-
+      
```

Comment:
This will have to be removed from the changeset, otherwise our tooling will block merge since we only allow `eng/common` changes to be made in the `azure-sdk-tools` repo.

Reply:
i need it otherwise i cannot pass parameters from the matrix to the resource yaml

Reply:
i'll figure out a way to pass things through additional params.

Reply:
done